app/src/org/commcare/provider/SimprintsCalloutProcessing.java

```diff
@@ -0,0 +1,104 @@
+package org.commcare.provider;
+
+import android.content.Intent;
+import android.util.Base64;
+import android.util.Log;
+
+import com.simprints.libsimprints.Identification;
+import com.simprints.libsimprints.Registration;
+
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.instance.AbstractTreeElement;
+import org.javarosa.core.model.instance.TreeReference;
+import org.javarosa.core.services.locale.Localization;
+import org.javarosa.core.util.OrderedHashtable;
+import org.odk.collect.android.jr.extensions.IntentCallout;
+
+import java.util.Collections;
+import java.util.Hashtable;
+import java.util.List;
+import java.util.Vector;
+
+/**
+ * Process simprints fingerprint registration and identification callout results.
+ *
+ * @author Phillip Mates (pmates@dimagi.com).
+ */
+public class SimprintsCalloutProcessing {
+    private static final String TAG = SimprintsCalloutProcessing.class.getSimpleName();
+
+    public static boolean isIdentificationResponse(Intent intent) {
```

Comment:
Can you add comments explaining what an identification response and a registration response are, respectively